src/main/java/nl/tudelft/contextproject/model/roomIO/RoomReader.java

```diff
@@ -1,45 +1,60 @@
-package nl.tudelft.contextproject.roomIO;
+package nl.tudelft.contextproject.model.roomIO;
 
 import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
 import java.io.File;
 import java.io.FileNotFoundException;
-import java.io.FileReader;
-import java.io.IOException;
 import java.util.List;
 import java.util.Set;
 
 import com.jme3.light.Light;
 
-import nl.tudelft.contextproject.files.FileUtil;
-import nl.tudelft.contextproject.model.Entity;
+import nl.tudelft.contextproject.util.FileUtil;
+import nl.tudelft.contextproject.model.entities.Entity;
 import nl.tudelft.contextproject.model.level.MazeTile;
 import nl.tudelft.contextproject.util.ScriptLoaderException;
 
 /**
  * Utility class for loading rooms/levels from file.
  */
 public final class RoomReader {
-	
-	private RoomReader() {};
+
+	/**
+	 * Private constructor to avoid instantiation.
+	 */
+	private RoomReader() {}
 	
 	/**
 	 * Import a room from the specified file.
 	 * Note that you can also use this method to load entire levels by using a (0,0) offset.
-	 * @param folder The folder to load the room from.
-	 * @param tiles The MazeTile array to store the loaded tiles in.
-	 * @param entities The set to add all the loaded entities to.
-	 * @param lights The list to add all the loaded lights to.
-	 * @param xOffset The horizontal offset that is used for moving all loaded items.
-	 * @param yOffset The vertical offset that is used for moving all loaded items.
-	 * @throws IOException When something goes wrong.
+	 *
+	 * @param folder
+	 * 		the folder to load the room from
+	 * @param tiles
+	 * 		the MazeTile array to store the loaded tiles in
+	 * @param entities
+	 * 		the set to add all the loaded entities to
+	 * @param lights
+	 * 		the list to add all the loaded lights to
+	 * @param xOffset
+	 * 		the horizontal offset that is used for moving all loaded items
+	 * @param yOffset
+	 * 		the vertical offset that is used for moving all loaded items
+	 * @throws IOException
+	 * 		when something goes wrong
 	 */
 	public static void importFile(String folder, MazeTile[][] tiles, Set<Entity> entities, List<Light> lights, int xOffset, int yOffset) throws IOException {
-		try (BufferedReader br = new BufferedReader(new FileReader(getMapFile(folder)))) {
+		try (BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(getMapFile(folder)), "UTF-8"))) {
```

Comment:
A few things.
1) Why was this changed from a FileReader?
2) Doesn't this throw a CharsetNotFound exception that has to be caught or something?

I think new FileReader(..., StandardCharsets.UTF_8) would be better (something like that).